docs/guides/about/intro.rst

```diff
@@ -36,7 +39,7 @@ the `Commonwealth Scientific and Industrial Research Organisation`_ (CSIRO) and
 Technology
 ----------
 
-Digital Earth Australia is powered by `Open Data Cube <http://opendatacube.org/>`_ (ODC), an open-source library for the management, analysis, and processing of Geographic Information System (GIS) data -- namely Earth Observation satellite data. It is a global initiative to increase the value and use of satellite data by providing users with access to free data management technologies and analysis platforms. At its core, ODC is a set of Python libraries and a PostgreSQL database that facilitate working with geospatial raster data.
+DEA is powered by `Open Data Cube <http://opendatacube.org/>`_ (ODC), an open-source library for the management, analysis, and processing of Geographic Information System (GIS) data — namely Earth Observation satellite data. It is a global initiative to increase the value and use of satellite data by providing users with access to free data management technologies and analysis platforms. At its core, ODC is a set of Python libraries and a PostgreSQL database that facilitate working with geospatial raster data.
 
-The ODC source code can be `found on GitHub <https://github.com/opendatacube/datacube-core>`.
+On GitHub, you will find the `ODC source code <https://github.com/opendatacube/datacube-core>`_.
```

Comment:
This new wording feels less direct and more passive - I prefer the original personally but it's a minor thing. 🙂

Reply:
I agree actually, it should be more 'active voice'. Ive reworded to:

```rst
The ODC source code can be found in the `datacube-core repository on GitHub <https://github.com/opendatacube/datacube-core>`_.
```